src/preconnect.js

```diff
@@ -109,19 +109,29 @@ class Preconnect {
   /**
    * Asks the browser to prefetch a URL. Always also does a preconnect
    * because browser support for that is better.
+   *
+   * If usePreloadIfSupported set to true and the browser supports preloading
+   * the URL will be preloaded instead - given higher priority and will be fetched
+   * as soon as possible.
+   *
    * @param {string} url
+   * @param {boolean=} opt_usePreloadIfSupported
    */
-  prefetch(url) {
+  prefetch(url, opt_usePreloadIfSupported) {
```

Comment:
No need for this arg. Lets always use it.

But do add an arg for the type. E.g. "html", "js", etc.